pages/my-va/index.md

```diff
@@ -1,5 +0,0 @@
----
-title: My VA
-layout: page-react.html
-entryname: dashboard
----
```

Comment:
Migrated here, https://github.com/department-of-veterans-affairs/vets-website/pull/10660/files#diff-377c2de5849b24c0c22549d9917ebafcR5.